landing-page/content/common/spark-quickstart.md

```diff
@@ -0,0 +1,379 @@
+if you wanted to merge in records from a table named `prod.nyc.updates`, you could run a `MERGE INTO` command to merge the updates into the `prod.nyc.taxis`
+table and simply add the `Fare_amount` for records where the `VendorID` and `Trip_ID` already exist.
+
+First, create the `updates` table.
+```sql
+CREATE TABLE prod.nyc.updates
+(
+  VendorID bigint,
+  TripID bigint,
+  Trip_distance float,
+  Fare_amount double,
+  Store_and_fwd_flag string
+);
+```
+Next, insert a few records.
+```sql
+INSERT INTO prod.nyc.updates
+VALUES (1, 1000371, 1.8, 0.50, 'N'), (3, 1000375, 8.9, 37.15, 'Y');
+```
+Finally, merge the `updates` table into the `taxis` table.
+```sql
+MERGE INTO prod.nyc.taxis t USING (SELECT * FROM prod.nyc.updates) u ON t.VendorID = u.VendorID and t.TripID = u.TripID
+WHEN MATCHED THEN UPDATE SET t.Fare_amount = t.Fare_amount + u.Fare_amount
+WHEN NOT MATCHED THEN INSERT *;
+```
+Querying the `taxis` table, you can see that the fare amount of `0.50` has been added to `15.32` for `VendorID = 1 AND TripID = 1000371`.
+```sql
+SELECT VendorID, TripID, Fare_amount
+FROM prod.nyc.taxis
+WHERE VendorID = 1 AND TripID = 1000371;
+```
+| VendorID | TripID  | Fare_amount |
+| -------- | ------- | ----------- |
+| 1        | 1000371 | 15.82       |
+
+{{< hint warning >}}
+Iceberg supports writing DataFrames using the new [v2 DataFrame write API](../spark-writes#writing-with-dataframes). The old `write` API is supported, but _not_ recommended.
+{{< /hint >}}
+
+## Adding Iceberg to your Existing Spark Environment
+
+To add Iceberg, use the `--packages` option.
+
+{{% codetabs "AddIcebergToSpark" %}}
+{{% addtab "SparkSQL" checked %}}
+{{% addtab "SparkShell" %}}
+{{% addtab "PySpark" %}}
+{{% tabcontent "SparkSQL"  %}}
+```sh
+spark-sql --packages org.apache.iceberg:iceberg-spark-runtime-3.2_2.12:{{% icebergVersion %}}
+```
+{{% /tabcontent %}}
+{{% tabcontent "SparkShell" %}}
+```sh
+spark-shell --packages org.apache.iceberg:iceberg-spark-runtime-3.2_2.12:{{% icebergVersion %}}
+```
+{{% /tabcontent %}}
+{{% tabcontent "PySpark" %}}
+```sh
+pyspark --packages org.apache.iceberg:iceberg-spark-runtime-3.2_2.12:{{% icebergVersion %}}
+```
+{{% /tabcontent %}}
+{{% /codetabs %}}
+
+{{< hint info >}}
+If you want to include Iceberg in your Spark installation, add the [`iceberg-spark-runtime-3.2_2.12` Jar](spark-runtime-jar) to Spark's `jars` folder.
+{{< /hint >}}
+
+[spark-runtime-jar]: https://search.maven.org/remotecontent?filepath=org/apache/iceberg/iceberg-spark-runtime-3.2_2.12/{{% icebergVersion %}}/iceberg-spark-runtime-3.2_2.12-{{% icebergVersion %}}.jar
+
+### Adding catalogs
+
+Iceberg comes with [catalogs](../spark-configuration#catalogs) that enable SQL commands to manage tables and load them by name. Catalogs are configured using properties under `spark.sql.catalog.(catalog_name)`.
```

Comment:
It nearly sounds like this is saying that Iceberg has pre-configured catalogs. You might want to change it to "Iceberg has several catalog back-ends that can be used to track tables, like JDBC, Hive MetaStore, Glue, etc."

Reply:
You may also want to mention that so far, the tutorial has used a local JDBC catalog. But you can use these instructions to configure another catalog.

Reply:
Done and done! Thanks for the wording suggestion.